packs/equipment-effects/effect-glowing-lantern-fruit-lantern-light.json

```diff
@@ -0,0 +1,89 @@
+{
+    "_id": "JCMACUq7FkEksV0Q",
+    "img": "systems/pf2e/icons/features/classes/lantern.webp",
+    "name": "Effect: Glowing Lantern Fruit (Lantern Light)",
```

Comment:
Since the Glowing Lantern Fruit can produce two effects, I appended the name of the specific one. Since the other one will likely never have an effect, I wasn't sure whether this was appropriate.